### PR TITLE
Update centralized-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by Alan1Mulhall**
Clarifying AAD requirements based on user feedback.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/centralized-deployment-of-add-ins.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Centralized deployment of add-ins requires that the users have Exchange Online a
     
     \*In MSI versions of Outlook, admin-installed add-ins will show in the appropriate ribbon in Outlook but doesn't show the add-in in 'My add-ins' section.
     
-- Your directory must be federated to Azure Active Directory.
+- Your tenant's directory must either be in or federated to Azure Active Directory.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
- Your tenant's directory must either be in, or federated to, Azure Active Directory.
```

**Comment by dianef77** [comment]
@Sirkkuw can you please review this one and decide whether to merge it?  Thanks!

**Comment by mypil** [comment]
@Sirkkuw - I would like to follow up on this. can you please review and merge the content of this PR? 

Thank you.

**Comment by mypil** [comment]
@Sirkkuw - I would like to follow up on this. can you please review and merge the content of this PR?

Thank you.

**Comment by Sirkkuw** [comment]
@mypil I am looking at this - it has a conflict so I might have to reject the PR and fix by hand. Will keep you advised.

**Comment by Sirkkuw** [comment]
@mypil I made the fixes, fixed his language, and also couple of PowerShell syntax fixes to the topic, I will close this one because there is a conflict.
See https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro-pr/pull/712

**Pull request closed without merging**